docs/what.rst

```diff
@@ -0,0 +1,29 @@
+=====================
+What is a Certificate?
+=====================
```

Comment:
This is a minor point, but an open source contributor recently went through our docs and cleaned up all the error messages you get when you build our docs (see #4706). These lines introduce new ones though because the title overline and underlines are too short. You should add an extra `=` character to each line.